foundation/foundation-mda/src/main/resources/templates/deployment/configuration-store/v2/configuration-store.values-dev.yaml.vm

```diff
@@ -9,13 +9,14 @@ aissemble-configuration-store-chart:
     deployment:
       image:
         imagePullPolicy: IfNotPresent
-      supplementalVolumes:
-        - name: configurations
-          hostPath:
-            path: /local/path/set/in/tilt
-            type: Directory
-      supplementalVolumeMounts:
-        - mountPath: /configurations
-          name: configurations
     service:
       type: LoadBalancer
+
+  configurationVolume:
+    enabled: true
+    name: "configuration-store"
+    storageType: "local"
+    volumePathOnNode: "/local/path/set/in/tilt"
```

Comment:
S: Recommend only adding the things that are changing the defaults. This gives us a bit more maneuvering room to change the default setup in the future without requiring projects' to manually make changes.  Only exception I'd make is for any values we _know_ will need to be set (at least in the vast majority of cases.) E.g. I might leave `storageType` and `storageClass` in the base values file template, even though they technically are using the default values.